launcher/CMakeLists.txt

```diff
@@ -83,18 +90,26 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 if(TARGET Qt6::Core)
 	qt_wrap_ui(launcher_UI_HEADERS ${launcher_FORMS})
+	if(ENABLE_TRANSLATIONS)
+		qt_add_translation( launcher_QM ${launcher_TS} )
+	endif()
 else()
 	qt5_wrap_ui(launcher_UI_HEADERS ${launcher_FORMS})
+	if(ENABLE_TRANSLATIONS)
+		set_source_files_properties(${launcher_TS} PROPERTIES OUTPUT_LOCATION ${CMAKE_CURRENT_BINARY_DIR}/translation)
```

Comment:
according to the [docs](https://doc.qt.io/qt-6/qtlinguist-cmake-qt-add-translations.html#options), this line should be used regardless of Qt version

Reply:
Done